tests/expectations/forward_declaration.c

```diff
@@ -0,0 +1,40 @@
+#if defined(CBINDGEN_STYLE_TYPE)
+/* ANONYMOUS STRUCTS DO NOT SUPPORT FORWARD DECLARATIONS!
```

Comment:
I tried several things to avoid the test for `Style::Type` to fail, but this was the only way to actually get it to work. Forward declaration just cannot work for that style, and this allows me to completely skip the compilation test - thus succeeding.